compiler/testdata/errors.go

```diff
@@ -13,18 +13,62 @@ func implementation() {
 
 type Uint uint32
 
+type S struct {
+	a [4]uint32
+	b uintptr
+	c int
+	d float32
+	e float64
+}
+
 //go:wasmimport modulename validparam
-func validparam(a int32, b uint64, c float64, d unsafe.Pointer, e Uint)
+func validparam(a int32, b uint64, c float64, d unsafe.Pointer, e Uint, f uintptr, g string, h *int32, i *S)
+
+// ERROR: //go:wasmimport modulename invalidparam: unsupported parameter type [4]uint32
+// ERROR: //go:wasmimport modulename invalidparam: unsupported parameter type []byte
+// ERROR: //go:wasmimport modulename invalidparam: unsupported parameter type struct{a int}
+// ERROR: //go:wasmimport modulename invalidparam: unsupported parameter type chan struct{}
+// ERROR: //go:wasmimport modulename invalidparam: unsupported parameter type func()
+//
+//go:wasmimport modulename invalidparam
+func invalidparam(a [4]uint32, b []byte, c struct{ a int }, d chan struct{}, e func())
+
+//go:wasmimport modulename validreturn1
+func validreturn1() int32
```

Comment:
Do we care about having these. named `validreturnInt32` or something so we don't just have numbers?